### ISSUE TITLE
Commenting on blogs and datasets

### ISSUE THREAD

**Issue by keeganmcbride** [okestonia/jkan#94]
We could integrate something like disqus to enable commenting and discussions on datasets, blogs, etc. That being said, I don't think it is available with Estonian text right now. Does it make sense to integrate this?

**Comment by infokujur** [comment]
Yes, we need to get really clear - who is watching what channel and then we can open new ones. Let's implement first what we have and then I'm opened to discuss next options.
